sdks/builds/setup-emsdk.sh

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
why use a separate sh file?

Reply:
The script was a lot more convoluted before. I'll merge.